examples/hello-world/step-by-step/higgs/prepare_data.ipynb

```diff
@@ -0,0 +1,363 @@
+    "\n",
+    "```\n",
+    "/tmp/nvflare/dataset/output/\n",
+    "\n",
+    "```\n",
+    "\n",
+    "To make it similar to the real world use cases, we put features (CSV file headers) into a file in the input directory.  When we split the file, we make sure each site will has a \"header.csv\" file corresponding to the csv data. In horizontal split. all the header will be the same. but for vertical learning, each site may have different headers. \n",
+    "\n",
+    "\n",
+    "We create a simple python code to split data: called split_csv.py. Let's run this, you will need to wait for few minutes. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "e9d5c919-9d99-494f-b216-8431ff2f23c8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "features written to /tmp/nvflare/dataset/input/headers.csv\n"
+     ]
+    }
+   ],
+   "source": [
+    "import csv\n",
+    "\n",
+    "# Your list of data\n",
+    "features = [\"label\", \"lepton_pt\", \"lepton_eta\", \"lepton_phi\", \"missing_energy_magnitude\", \"missing_energy_phi\", \"jet_1_pt\", \"jet_1_eta\", \"jet_1_phi\", \"jet_1_b_tag\", \"jet_2_pt\", \"jet_2_eta\", \"jet_2_phi\", \"jet_2_b_tag\", \"jet_3_pt\", \"jet_3_eta\", \"jet_3_phi\", \"jet_3_b_tag\",\\\n",
+    "            \"jet_4_pt\", \"jet_4_eta\", \"jet_4_phi\", \"jet_4_b_tag\", \\\n",
+    "            \"m_jj\", \"m_jjj\", \"m_lv\", \"m_jlv\", \"m_bb\", \"m_wbb\", \"m_wwbb\"]\n",
+    "\n",
+    "# Specify the file path\n",
+    "file_path =  '/tmp/nvflare/dataset/input/headers.csv'\n",
+    "\n",
+    "with open(file_path, 'w', newline='') as file:\n",
+    "    csv_writer = csv.writer(file)\n",
+    "    csv_writer.writerow(features)\n",
+    "\n",
+    "print(f\"features written to {file_path}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "33eb485b-f28b-42aa-935e-3bdcd7087047",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "label,lepton_pt,lepton_eta,lepton_phi,missing_energy_magnitude,missing_energy_phi,jet_1_pt,jet_1_eta,jet_1_phi,jet_1_b_tag,jet_2_pt,jet_2_eta,jet_2_phi,jet_2_b_tag,jet_3_pt,jet_3_eta,jet_3_phi,jet_3_b_tag,jet_4_pt,jet_4_eta,jet_4_phi,jet_4_b_tag,m_jj,m_jjj,m_lv,m_jlv,m_bb,m_wbb,m_wwbb\n"
+     ]
+    }
+   ],
+   "source": [
+    "!cat /tmp/nvflare/dataset/input/headers.csv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "965d0283-53dc-48e6-86bb-605c5b84ebd9",
+   "metadata": {},
+   "source": [
+    "Now we prepare to split data, note that, we used certain sample rate to make demo faster to run. You can change the number to even smaller such 0.003 to reduce the file size especially when you development or debugging. "
```

Comment:
suggest just: "Now we are ready to split the data"

Suggest move the "note" part right in front of where we call the "split_csv.py" or as a comment in that code block.